src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -49,9 +49,9 @@ private FlutterSdkUtil() {
   /**
    * Return the environment variable value to use when shelling out to the Flutter command-line tool.
    */
-  public static String getFlutterEnvValue() {
+  public static String getFlutterHostEnvValue() {
     final String clientId = ApplicationNamesInfo.getInstance().getFullProductName().replaceAll(" ", "-");
-    final String existingVar = System.getenv(FLUTTER_ENVIRONMENT);
+    final String existingVar = System.getenv(FLUTTER_HOST_ENV);
     return existingVar == null ? clientId : (existingVar + ":" + clientId);
```

Comment:
So, I think we're ok passing plain text here. We did do something very similar for invoking pub a few weeks ago. I don't think this graduate to a full user agent type thing, but just be used in analytics to get a better sense of how the flutter cli is used (exclude bots, only human cli usages and not tool driven, ...).